src/augment/augment.js

```diff
@@ -310,6 +310,7 @@ export const shouldAugmentRelationshipField = (
 const APIConfiguration = {
   QUERY: 'query',
   MUTATION: 'mutation',
+  SUBSCRIPTION: 'subscription',

```

Comment:
@michaeldgraham I'm testing this in various cases and I notice that the Subscription type augmentation is added by default (when using `makeAugmentedSchema`) unless `{config: {subscription: false}}` is explicitly specified.

It looks like the default config object (for the `makeAugmentedSchema` path) is set [in augment.js here](https://github.com/neo4j-graphql/neo4j-graphql-js/blob/master/src/augment/augment.js#L320-L328) 

```
/**
 * Builds the default values in a given configuration object
 */
export const setDefaultConfig = ({ config = {} }) => {
  const configKeys = Object.keys(config);
  Object.values(APIConfiguration).forEach(configKey => {
    if (!configKeys.find(providedKey => providedKey === configKey)) {
      config[configKey] = true;
    }
  });
  return config;
};
```

which looks like it is setting the default config to true for all `APIConfiguration` values (which includes subscription). For the others (query, mutation, temporal, spatial) a default value of true makes sense but I think `config.subscription` needs an explicit default value of `false` here?

Reply:
Ah - I wasn't sure, as we have query and mutation defaults as true so I was trying to be consistent with defaults for GraphQL operations generally, but as subscriptions are somewhat of a unique operation and may not be needed or wanted initially, I'll go ahead and change the default~

Reply:
Yeah, I think a default value of false for subscriptions makes sense, especially as the user is providing the `publish` and `subscribe` functions as part of `config.subscription`

Reply:
Yeah - I didn't generate defaults for those abstract publish and subscribe functions either, as although it would be nice to do so, one might want to use a pubsub implementation other than that from apollo server. I just committed the change~

Reply:
Whoops, need to update the augmentation tests now given the change to defaults~

Reply:
Looks all green~